tests/cover/test_manual_given_invocation.py

```diff
@@ -64,7 +64,16 @@ def test_argspec_lines_up(f, g):
 
 def test_converts_provided_kwargs_into_args():
     @given(hello=int, world=int)
-    def greet(**kwargs):
+    def greet(hello, **kwargs):
```

Comment:
Nitpick: I think this test now tests exactly the opposite of what it used to test so should probably have a name change.